### ISSUE TITLE
Weird password reset message

### ISSUE THREAD

**Issue by natt-tester** [GluuFederation/oxTrust#1152]
It's possible to get this weird message when you click on the reset link in the email when you've already used it:

<img width="1363" alt="ubuntu16_gserver3 1 4_wrong_password_success" src="https://user-images.githubusercontent.com/38302725/44651254-350b0e00-aa2c-11e8-8794-9996cad1d5aa.png">

**Ideally, upon clicking the already used link we should have a message and a field to re-enter your e-mail to generate a new link:**

"Oops!
This link has already been used to reset your password. 
Re-enter your e-mail to generate a new link."

But for now we can skip the field part and just change the strings.

**Comment by willow9886** [comment]
@natt-tester how about :

> Oops! This link has expired. Please try again.

**Comment by natt-tester** [comment]
@willow9886 yes, that will work for both the expired and already used e-mail. Btw, how long is the e-mail valid?

**Comment by syntrydy** [comment]
![image](https://user-images.githubusercontent.com/7513418/44913738-21122580-ad26-11e8-8e40-e38bdedb0e20.png)

**Comment by syntrydy** [comment]
I think it is better than **Oops ....**. The user can generate new link directly if needed.

**Comment by syntrydy** [comment]
![image](https://user-images.githubusercontent.com/7513418/44916918-c630fc00-ad2e-11e8-9c8e-117c36e77dfe.png)

**Comment by syntrydy** [comment]
![image](https://user-images.githubusercontent.com/7513418/44917247-c8e02100-ad2f-11e8-8a34-7a6793c32482.png)

**Comment by willow9886** [comment]
@syntrydy can you update the content? 

> The reset link is no more valid.... 

This can be improved. 

You can simply say:

> The reset link is no longer valid. Re-enter your email to generate a new link.

**Comment by sahiliamsso** [comment]
Not able to reset user password, After clicking on the link, It redirects to change password screen but when I hit Update Password, see this error
![image](https://user-images.githubusercontent.com/21104811/45352916-3402e080-b5d7-11e8-9efd-586677f83b1a.png)

**Comment by sahiliamsso** [comment]
As the message says “The reset link is no more valid.It has already been used to reset your password or it has expired. Re-enter your e-mail to generate a new link.”

What is the default  expiry time? Can it be configured in oxtrust (say 15min/1hr/24hrs)
I don’t see it mentioned in the password reset email.

**Comment by syntrydy** [comment]
The password reset is working as expected in the current code. 
Note that I use the issue title and his description to fix it. So think like** specified the expiration time for reset link will be a separate issue.

**Comment by syntrydy** [comment]
https://github.com/GluuFederation/oxTrust/issues/1200

**Comment by natt-tester** [comment]
We need to improve the messages. I think we agreed on pop-ups with success messages "Reset password e-mail has been sent" and "Reset password successful" on the login screen, but we don't have them in RC7.
After changing the password, the user is redirected to the login page. The messages are visible only when you go back to the reset password page, which isn't good:
 
<img width="528" alt="no_longer" src="https://user-images.githubusercontent.com/38302725/45999709-c7fd9d80-c0e9-11e8-824b-a120584d1115.png">
- please change "more" to "longer"

<img width="479" alt="invalid_link" src="https://user-images.githubusercontent.com/38302725/45999742-ea8fb680-c0e9-11e8-849f-c53a6c702219.png">
- disable the first message with the caps
